packages/pyright-internal/src/analyzer/tracePrinter.ts

```diff
@@ -260,7 +260,8 @@ export function createTracePrinter(roots: string[]): TracePrinter {
             return printType(o as Type);
         }
 
-        assertNever(o);
+        // Do nothing, we can't print it.
+        return '';

```

Comment:
I should have put a comment here. 

The logging function doesn't pass the parameters to itself anymore so it attempts to log all parameters. Not all parameter types are supported, so when that happens instead of asserting, just do nothing.